modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/Ocl.java

```diff
@@ -26,7 +27,7 @@
 @Valid
 @Data
 @Slf4j
-public class Ocl {
+public class Ocl implements Serializable {
 
```

Comment:
Add serialVersionUID to avoid potential incompatibilities during deserialization.